stackle_api/server.js

```diff
@@ -1,6 +1,7 @@
 require('dotenv').config();
 const express = require('express');
 const app = express();
+<<<<<<< HEAD
```

Comment:
There's another conflict here. Get in touch with @shivamarora96 when fixing API related conflicts. It will be better that way